enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -971,13 +971,25 @@ func (e *partitionEvictor) computeSize() (int64, int64, int64, error) {
 		})
 	}
 
+	acPrefixRange := func(start, end []byte) ([]byte, []byte) {
+		left := make([]byte, 0)
+		left = append(left, e.acPrefix...)
+		left = append(left, start...)
+
+		right := make([]byte, 0)
+		right = append(right, e.acPrefix...)
+		right = append(right, end...)
+		return left, right
```

Comment:
nit: do you want to pre-compute the capacity for left and right?

Reply:
Done.